server/src/app.controller.ts

```diff
@@ -1,20 +1,34 @@
-import { Controller, Get, Inject, Logger, LoggerService } from '@nestjs/common';
-import { AppService } from '@/app.service';
-import { ApiOperation, ApiTags } from '@nestjs/swagger';
-
-@ApiTags('기본')
-@Controller()
-export class AppController {
-  constructor(
-    private readonly appService: AppService,
-    @Inject(Logger) private readonly logger: LoggerService,
-  ) {}
-
-  @ApiOperation({ summary: '기본' })
-  @Get()
-  getHello(): any {
-    this.logger.log('Hello??');
-    return { result: 123 };
-    // return this.appService.getHello();
-  }
-}
+import { Controller, Get, Inject, Logger, LoggerService } from '@nestjs/common';
+import { AppService } from '@/app.service';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { RedisClientType } from 'redis';
+import { REDIS_CLIENT } from '@/common/redis/redis.module';
+
+@ApiTags('기본')
+@Controller()
+export class AppController {
+  constructor(
+    private readonly appService: AppService,
+    @Inject(Logger) private readonly logger: LoggerService,
+    @Inject(REDIS_CLIENT) private readonly redisClient: RedisClientType,
+  ) {}
+
+  @ApiOperation({ summary: '기본' })
+  @Get()
+  getHello(): any {
+    this.logger.log('Hello??');
+    return { result: 123 };
+    // return this.appService.getHello();
+  }
+
+  @Get('/redis')
+  async testRedis(): Promise<any> {

```

Comment:
```suggestion
  async testRedis() {

```

은태님! 타입스크립트 자체에서 반환 타입을 자동으로 `Promise<any>`로 설정할텐데,
꼭 명시적으로 `Promise<any>`를 쓸 이유가 있을까 생각됩니다!

반환 타입을 제거하거나, `Promise<type>`과 같이 어떤 타입이 반환될지 명시해주시면 좋을 것 같습니다!

Reply:
제가 ts 초보이기도 하고, 테스트 용도라서 any라도 명시해주는게 맞다고 생각했습니다!
나중에 제대로 된 로직 짤 때 타입 지정해줄 예정입니다!